src/modules/base/header/header.js

```diff
@@ -38,6 +42,34 @@
             }
         });
 
+        init();
+
+        function init(){
+            _getUserName();
+            _getUsersList();
+
+        }
+
+        function _getUserName(){
+            if (!userStore.users.length) {
+                userStore.getUserInfo()
+                    .then(function(data) {
+                        vm.isDataLoading = false;
```

Comment:
@a2batic I think "vm.isDataLoading = false" is not required here. Remove it if not required.